tools/mpremote/mpremote/commands.py

```diff
@@ -555,7 +555,7 @@ def _do_romfs_deploy(state, args):
     romfs_filename = args.path
 
     # Read in or create the ROMFS filesystem image.
-    if romfs_filename.endswith(".romfs"):
+    if os.path.isfile(romfs_filename):
```

Comment:
My intention here was to eventually support other extensions to mean different things.  Eg `.json` could be a JSON manifest of files to turn into a ROMFS, and `.py` could be a standard MicroPython manifest file.  This change will no longer allow any of that, we'd need to find a different way to specify the type of the file.

Reply:
How about `.img` for ready to deploy images? It's seems standard for filesystem images? 

https://en.wikipedia.org/wiki/IMG_(file_format)

Reply:
What's wrong with `.romfs`?

(Or, could possibly support both `.romfs` and `.img` if it's really necessary.)

Reply:
I updated it, but when/if other file types are supported, wouldn't something like the following be needed anyway? So perhaps leave `isfile` ?

```Python
    if romfs.startswith(VfsRomWriter.ROMFS_HEADER):
       # romfs image ready to deploy
    elif extension == "json":
     # parse json
    elif extension == "py":
     # python
    else:
     raise(...) 
```

> What's wrong with .romfs?

Nothing really, I just call the images `romfs<part>` so they'll end up `romfs0.romfs` which is just not as nice as `romfs0.img`.

Reply:
> I updated it, but when/if other file types are supported, wouldn't something like the following be needed anyway? So perhaps leave `isfile` ?

Yes, it's OK to leave `isfile` in.  But also check the extension, maybe just by using `romfs_filename.endswith((".img", ".romfs"))`?

Reply:
Updated!